pkg/storage/unified/README.md

```diff
@@ -269,6 +269,23 @@ You can then list the previously-created playlists with:
 kubectl --kubeconfig=./grafana.kubeconfig get playlist
 ```
 
+### Everything together: run a separate database and a GRPC server
+
+This is the current default when running unified storage.
+Apart of making sure your configs match what is present in sections "Use a separate database" and "Start GRPC storage-server", to start the server you can run
```

Comment:
Might be clearer to say:

_Add the following to your config.ini_
```ini
[grafana-apiserver]
storage_type = unified-grpc
```

Or else `make run` won't connect to US.